angular-library/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ### 4.6.0
 
+> (insert date here upon release)
```

Comment:
If you want to leave this out you can, and when we do the release we can copy the changelogs over to the other libraries so we only have to do it once. Also it looks like this one is behind, so we will want to fix that when we do the release too!